ckanext/canada/cli.py

```diff
@@ -320,6 +328,9 @@ def _copy_datasets(source_datastore_uri: Optional[Union[str, None]], user: Optio
 
         packages = iter(sys.stdin.readline, '')
         for package in packages:
+            sys.stdout.write(json.dumps(['package_id', 'action', 'reason', 'error', 'failure_reason', 'failure_trace', False]) + '\n')
+            sys.stdout.flush()
+            continue
```

Comment:
unconditionally skip all the rest of the code in this loop?

Reply:
oh damn my debug code MY BAD